polylogue/core/config_validation.py

```
@@ -0,0 +1,55 @@
+    class Config:
+        extra = "forbid"
+
+
+class ConfigModel(BaseModel):
+    defaults: Optional[DefaultsModel] = None
+
+    class Config:
+        extra = "forbid"
+
+    @model_validator(mode="after")
+    def ensure_theme(cls, values: "ConfigModel") -> "ConfigModel":
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)